### PR TITLE
brake lights

### LINKED ISSUE #17: [Feature Request]: Brake lights when the robot initiates a brake
# Expected Behavior
I expect the robot to show brake lights to indicate that it is deaccelerating to its surroundings.

# Current Behavior
Currently there is no such thing as a brake light. When the robot stops, it does not give any audio or visual feedback

### PR DESCRIPTION
**Pull request by Quincccy**
# Description

lights when braking

Fixes #17 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

With the BoeBot

# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/vehicle/DrivingLights.java [hunk 1]
```diff
@@ -9,29 +9,38 @@
...
     }
 
     /**
-     * Returns if the given direction is neutral and otherwise changes the
-     * direction attribute in the given direction.
      * @param direction - The direction that the Bot is currently heading.
      */
     public void start(Direction direction) {
-        if (this.direction == direction) {
+        if(this.direction == direction){
```
Comment (Daantuk01) [inline src/vehicle/DrivingLights.java:43]: spaties bij if en )

File: src/vehicle/DrivingLights.java [hunk 2]
```diff
@@ -43,12 +52,31 @@ public void start(Direction direction) {
      */
     public void on() {
         if (this.direction == Direction.FORWARD) {
+            if(this.backLightsOn == true || this.brakeLightsOn == true) {
```
Comment (Daantuk01) [inline src/vehicle/DrivingLights.java:55]: spatie bij if

File: src/vehicle/DrivingLights.java [hunk 3]
```diff
@@ -43,12 +52,31 @@ public void start(Direction direction) {
...
+                this.off();
+            }
             topRight.setColor(Color.WHITE);
             topLeft.setColor(Color.WHITE);
+            topMiddle.setColor(Color.WHITE);
+            this.frontLightsOn = true;
         }
         else if (this.direction == Direction.BACKWARD) {
+            if(this.frontLightsOn == true || this.brakeLightsOn == true) {
```
Comment (Daantuk01) [inline src/vehicle/DrivingLights.java:64]: spatie bij if

**Review by Daantuk01 — CHANGES_REQUESTED** [review]
spaties bij de if statements

**Review by Daantuk01 — DISMISSED** [review]

**Review by Momin2004 — APPROVED** [review]

**Pull request merged**